services/QuillLMS/client/app/bundles/Teacher/containers/EditOrCreateSubscription.jsx

```diff
@@ -21,33 +22,11 @@ export default class EditOrCreateSubscription extends React.Component {
     this.state = {
       subscription: defaultSubscription,
       firstFocused: false,
-      secondFocused: false
+      secondFocused: false,
+      schools: this.props.schools,
```

Comment:
Not important but ideally this would be deconstructed (you'd add `schools` to line 14 above and then just do:

```
  secondFocused: false,
  schools
}
```

here.